src/Stripe.net/Services/PaymentIntents/StripePaymentIntentTransferDataOptions.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripePaymentIntentTransferDataOptions : INestedOptions
+    {
+        [JsonProperty("transfer_data[amount]")]
```

Comment:
I'll never understand why we need to repeat the parent property's parameter name. Haven't tested either as the test suite would require a custom account and all that.